### PR TITLE
Apply variables to ensure the spacing is properly applied to 'cancel' button on Space settings general tab

### PR DESCRIPTION
**Pull request by luixxiul**
|Before|After|
|---------|------|
|before|after|

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: res/css/views/dialogs/_SpaceSettingsDialog.scss [hunk 1]
```diff
@@ -78,7 +78,6 @@ limitations under the License.
     .mx_AccessibleButton_hasKind {
         &.mx_AccessibleButton_kind_link {
             font-size: $font-14px;
-            margin: 7px 18px;
```
Comment (luixxiul) [inline res/css/views/dialogs/_SpaceSettingsDialog.scss:81]: The style of `Upload` link button next to the avatar is specified with `_SpaceBasicSettings.scss` so removing this line does not change that style.

**Review by kerryarchibald — CHANGES_REQUESTED** [review]
Both other settings (user and room) line up the cancel button directly to the left of the container:

Room:
Screenshot 2022-06-16 at 18 03 38
Profile:
Screenshot 2022-06-16 at 18 03 21

I think a better change would be to align the space settings design with room and user.

**Comment by luixxiul** [comment]
@kerryarchibald Is it OK to change the current UI? Please note that this PR does not change the current UI.

**Comment by luixxiul** [comment]
Closing due to no response for a very long time.

**Pull request closed without merging**